CHANGELOG.md

```diff
@@ -1,3 +1,131 @@
+# 0.26.0 (2018-08-??)
+
+## Language changes
+
+- **(breaking-change)** Revert do not collapse unions for sibling types. ([#6351], thanks @asterite)
+- **(breaking-change)** Restrict `exp.@var` to types in the same hierarchy. ([#6358], thanks @asterite)
+- **(breaking-change)** Constant lookup context in macro is now lexical. ([#5354], thanks @MakeNowJust)
+- **(breaking-change)** Evaluate instance var initializers at the metaclass level (ie: disallow using `self`). ([#6414], thanks @asterite)
+- Add `&+` `&-` `&*` `&**` operators parsing. NB: No behaviour is assigned to these operators yet. ([#6329], thanks @bcardiff)
+- Add support for empty `case` without `when`. ([#6367], thanks @straight-shoota)
```

Comment:
We could precise the main purpose of this: ... "to generate them with macros"

Reply:
An explanation for any change can be found in the respective PRs. It's unnecessary to mention here.